views/sponsor.ejs

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+<head>
+    <% include template/head.ejs %>
+    <link rel="stylesheet" type="text/css" href="/css/index.css"/>
+    <style>
+        .main-footer {
```

Comment:
Generally, we want to add styles to style.css file instead of in page for easier maintenance

Reply:
Also, this is not needed since it's in the style.css file already

Reply:
I added these lines in order to override the CSS for this page so that the footer will always stick on the bottom of the screen/browser.

Without these lines, the web page will look like this:
<img width="1414" alt="Screen Shot 2019-10-31 at 8 04 20 PM" src="https://user-images.githubusercontent.com/5493642/67995666-a4285780-fc19-11e9-9920-447df6651d47.png">